packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -173,6 +177,11 @@ abstract class CachedArtifact {
   }
 
   Future<void> update() async {
+    // Clear any existing artifact files.
+    if (file.parent.existsSync()) {
+      await file.parent.delete(recursive: true);
```

Comment:
Any reason why we're using `delete` (async) instead of `deleteSync`?

Reply:
If this takes any amount of time, the async option makes sure any progress indicators or other updates don't lock up